### DISCUSSION TITLE
How to get rid of the orange directory and customization

### DISCUSSION THREAD

**Discussion by neuromante** [SpiralLinux/SpiralLinux-project#370] [Q&A] [answered]
Hi everyone! Just installed Spiral KDE and upgraded to Testing. I can’t get rid of the dolphin orange icon on the taskbar despite the theme I chose. I just want my standard blue from breeze theme. Even the calendar widget has a strange background color. I want the plain breeze theme applied over all the desktop. Any hints?

**Comment by geckolinux** [comment]
Hi there, did you try setting the Plasma global theme to Breeze?

**Comment by olafdressler** [comment]
Hello everyone, first of all: @geckolinux a big thank you for this spin. I appreciate it.
I have the same wish. I like the twilight theme of breeze, but in the taskbar you have made a red color. When i set the normal breeze-theme, the color changed to the breeze-blue. But when i change to twilight back, the red colors in the taskbar returns (in the dark-breeze-theme, too).

**Reply by geckolinux** [reply, marked as answer]
Hi there, Plasma theming is a bit of a mess in the older version of Plasma that Debian Stable uses. To get rid of the red theming in the taskbar it looks like you need to:
1. In **Appearance > Plasma Style** delete **Breeze Dark Orange**
2. In **Appearance > Colors** delete **Numix**
3. In **Global Theme** select "Breeze Twilight"
4. You might need to log out and log in again to make everything fully apply.

**Reply by olafdressler** [reply]
Thanks...that works.

**Reply by olafdressler** [reply]
Now i've the problem with the color with firefox. Everytime it started with these orange color (highlighting, borders of fields ect.). Have already installed flatpak and deb from FF-Website, but everytime it started with these orange colors. The ".mozilla-Folder" in my home i have deleted before launch FF.

**Reply by geckolinux** [reply]
Hi there, in Plasma the theme for normal GTK applications (and Firefox) installed from DEB needs to be customized separately in **Appearance > Application Style > Configure Gnome/GTK Application Style...**.

**Reply by olafdressler** [reply]
Hi,
one thing at least (i hope ;-)):
How can i change the Theme of root? Because of my personal Plasma-theming (Nordic from eliverlara) and the Icon-Theming (Breeze-dark), the icons of the root-applications (e.g. synaptic) were bright.
I want to set the root-Theme to Breeze normal.

**Reply by geckolinux** [reply]
Hi there @olafdressler, the easiest solution is to login to Plasma as `root` and make the desired changes, which will also be applied when you run applications as `root` from your normal user account.
